Automattic/jetpack
# Jetpack: log event when converting video block to the new VideoPress video block

Fixes https://github.com/Automattic/jetpack/issues/29069

## Proposed changes:

* Jetpack: log event when converting video block to the new VideoPress video block

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



Yes. This PR logs the `jetpack_editor_videopress_block_manual_conversion_click` even the user converts manually current video block instances (v5) to the new VideoPress video block ones (v6)

## Testing instructions:






* Activate Jetpack plugin / activate VideoPress module
* Go to the block editor
* Set debug() scope with `dops:analytics`:

```
localStorage.setItem( 'debug', 'dops:analytics*' )
```
* Hard refresh
* Create a new v5 instance. Duplicate a few times (five it's fine 😃 )
* Open the block sidebar
* Convert the block by clicking on the `Transform blocks to VideoPress` button
* Confirm the app logs one event for every block converted.

<img width="892" alt="image" src="https://user-images.githubusercontent.com/77539/220354120-9ac69f14-d691-426d-b0ab-93f86f1a2aec.png">

* Confirm the `t` gif image is loaded

<img width="861" alt="Screen Shot 2023-02-21 at 10 41 47" src="https://user-images.githubusercontent.com/77539/220361934-92bd336f-e938-4ab6-9fba-d574a6bb2064.png">
